src/components/LandUseVocabulary.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import _ from 'lodash'
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger'
+import Tooltip from 'react-bootstrap/Tooltip'
+import Button from 'react-bootstrap/Button'
+
+import terms from '../assets/keywords'
+
+import './LandUseVocabulary.css';
+
+class LandUseVocabulary extends Component {
+  
+  state = {
+    usesString: this.props.string,
+  };
+
+  render() {
+    const { usesString } = this.state;
+    // splitting the uses
+    let uses = _.split(usesString, ',')
+    //return <div dangerouslySetInnerHTML={this.createMarkup(this.state.html)} />;
+    return uses.map(use=>this.renderUse(use))
+    return 'gal'
+  }
+  renderUse(use) {
+    let useTerm = this.finduse(use);
+
+    if(useTerm && useTerm.description){
+        return <OverlayTrigger
+            key={use}
+            overlay={
+            <Tooltip id={`tooltip-${use}`}>
+             <strong>{useTerm.title}</strong> <br></br>
+              {useTerm.description}
+            </Tooltip>
+          }
+        >
+        <Button variant="light">{use}</Button>
```

Comment:
Why are you using a button for something that isn't a button? Use another element for this.

Reply:
because it looks like the best practise for "tags" and land use is a tag